packages/calcite-components/src/components/meter/resources.ts

```diff
@@ -0,0 +1,14 @@
+export const CSS = {
+  meter: "meter",
```

Comment:
I think we can also simplify naming/classes here by dropping meter since it's the main context for this object.